src/main/db/controller/interface.ts

```diff
@@ -3,8 +3,10 @@
  */
 
 export interface ISearchOptions {
-    gt: unknown;
-    lt: unknown;
+    gte?: unknown;
```

Comment:
You can take options from here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/levelup/index.d.ts#L91

Reply:
yarn add @types/levelup